<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Hist</class>
 <widget class="QWidget" name="Form_Hist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_rgb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_r">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_g">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_b">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
